tests/perfalert/test_alert_modification.py

```diff
@@ -141,27 +141,27 @@ def test_improvement_summary_status_after_reassigning_regression(
         has_subtests=False,
         last_updated=datetime.datetime.now(),
     )
-    s = test_perf_alert_summary
 
-    create_perf_alert(
-        summary=s,
+    alert = create_perf_alert(
+        summary=test_perf_alert_summary,
         series_signature=signature1,
         is_regression=False,
         status=PerformanceAlert.ACKNOWLEDGED,
     )
 
-    assert s.status == PerformanceAlertSummary.IMPROVEMENT
+    assert test_perf_alert_summary.status == PerformanceAlertSummary.IMPROVEMENT
 
     create_perf_alert(
-        summary=s,
+        summary=test_perf_alert_summary_2,
         series_signature=signature2,
         is_regression=True,
-        related_summary=test_perf_alert_summary_2,
+        related_summary=test_perf_alert_summary,
```

Comment:
Here we are creating a new alert which is reassigned to our summary, this means that `related_summay ` contained the wrong summary. This is correcting that mistake.